picard/ui/options/plugins.py

```diff
@@ -246,30 +262,35 @@ def add_plugin_item(self, plugin, item=None):
         else:
             item.setText(1, plugin.version)
 
-        label = None
         if plugin.can_be_updated:
-            label = _("Update")
+            self.ui.plugins.headerItem().setText(3, _("Update"))
+            button = QtWidgets.QPushButton(_("Update"))
+            button.setMaximumHeight(button.fontMetrics().boundingRect(_("Update")).height() + 7)
+            self.ui.plugins.setItemWidget(item, 3, button)
+
+            def download_button_process():
+                self.ui.plugins.setCurrentItem(item)
+                self.download_plugin()
+            button.released.connect(download_button_process)
         elif plugin.can_be_downloaded:
-            label = _("Install")
+            button = QtWidgets.QPushButton(_("Install"))
             item.setFlags(item.flags() ^ QtCore.Qt.ItemIsUserCheckable)
-
-        if label is not None:
-            button = QtWidgets.QPushButton(label)
-            button.setMaximumHeight(button.fontMetrics().boundingRect(label).height() + 7)
+            button.setMaximumHeight(button.fontMetrics().boundingRect(_("Install")).height() + 7)
```

Comment:
There was a reason for `label = _("Install")` please keep it and don't repeat yourself.